nixos/modules/services/hardware/scanservjs.nix

```
@@ -0,0 +1,140 @@
+{ config
+, lib
+, pkgs
+, ... }:
+
+let
+  cfg = config.services.scanservjs;
+  settings = {
+    scanimage = "${pkgs.sane-backends}/bin/scanimage";
```

It's probably better to use the same backend as the sane service in case the user changes it.

```suggestion
    scanimage = "${config.hardware.sane.backends-package}/bin/scanimage";
```

---

done!